nservicebus/messaging/databus.md

```diff
@@ -68,7 +68,22 @@ snippet:MessageWithLargePayloadUsingConvention
 
 NServiceBus DataBus implementations currently behave differently with regard to cleanup of physical attachments used to transfer data properties. `FileShareDataBus` **does not** remove physical attachments once the message is gone. `AzureDataBus` **does** remove Azure storage blobs used for physical attachments once the message is gone.
 
+### Custom cleanup strategy
```

Comment:
Is the custom cleanup strategy only apply to FileShareDataBus? 
Should we change the first paragraph into two sections, i.e. FileShareDataBus and AzureDataBus and spell out the behavior differences?